docker/nginx.conf

```diff
@@ -128,15 +129,15 @@ http {
   }
 
   server {
-    listen [::]:8080;
-    listen 8080;
+    listen [::]:9999;
+    listen 9999;
 
     server_name _;
 
     # Path for static files
     root /usr/share/nginx/html;
 
-    location /status {
+    location /nginx_status {
```

Comment:
why'd you change this?

Reply:
all of the changes here were copied over from the monitor-ci repo. If you'll recall, all of the old nginx.conf files were deleted. This shows up as a diff in github because the file name is the same (i imagine)